runtime/fastly/builtins/fetch/request-response.cpp

```diff
@@ -238,17 +238,17 @@ uint32_t RequestOrResponse::handle(JSObject *obj) {
 }
 
 bool RequestOrResponse::has_body(JSObject *obj) {
-  MOZ_ASSERT(is_instance(obj));
+  // MOZ_ASSERT(is_instance(obj));
```

Comment:
Could you either add a TODO to each commented out assertion, or remove them? It's fine to leave them with a TODO if the intent is to fix them, but I'd rather we remove them entirely otherwise.

Reply:
Sure, added.